Core/src/org/sleuthkit/autopsy/modules/interestingitems/FilesSet.java

```diff
@@ -737,6 +755,46 @@ public boolean passes(AbstractFile file) {
 
         }
 
+        /**
+         * A class for checking whether a file's creation or modification
+         * occured in a specific range of time
+         */
+        static final class DateCondition implements FileAttributeCondition {
+
+            private final static long DAY_IN_S = 60 * 60 * 24;
```

Comment:
Maybe rename DAY_IN_S to something more obvious - SECS_PER_DAY?

Reply:
Good suggestion, thanks.